### DISCUSSION TITLE
[Projects Beta] Filter by assignment

### DISCUSSION THREAD

**Discussion by TolleyLikesRice** [community/community#17018] [Projects and Issues] [answered]
Would be really nice to filter the project view just to see what is assigned to me, or someone else.

**Comment by mattcosta7** [comment, marked as answer]
https://docs.github.com/en/issues/trying-out-the-new-projects-experience/filtering-projects

we support this with `assignee:@me` or `assignee:login`

**Reply by TolleyLikesRice** [reply]
I'm just stupid, thanks
